pf/tests/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/pulumi/pulumi-terraform-bridge/pf/tests
 
-go 1.21.12
+go 1.22
+
+toolchain go1.23.0
```

Comment:
Do we need this one? I've seen that cause issues before.

Reply:
They were added automatically, but it seems like I can remove them.